### PR TITLE
148 migrate extensions marketplace docs

### PR DESCRIPTION
**Pull request by billy-the-fish**
Added the extensions and filtered for platform.

### CODE REVIEW

File: extensions-marketplace/develop/integrate/banuba.mdx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+export const toc = [{}];
+
+This guide is provided by Banuba. Agora is planning a documentation upgrade program for all extensions on the marketplace. Please stay tuned.
+
+By integrating Banuba Face AR SDK and Agora.io SDK you can enable augmented reality features in video calls such as face filters, face touch up filters and virtual backgrounds. You can find the integration examples below.
+<PlatformWrapper platform="web">
+This extension is not provided for Web.
+</PlatformWrapper>
+<PlatformWrapper notAllowed="web">
```
Comment (dengyaxi) [inline extensions-marketplace/develop/integrate/banuba.mdx:17]: Add macOS and Windows in `notAllowed`. This extension is for Android and iOS only.

File: extensions-marketplace/develop/integrate/faceunity.mdx [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+description: >
+    Integrate and use the FaceUnity AR Filter extension in your  Android or iOS app.
+---
+export const toc = [{}];
+
+This page introduces how to integrate and use the FaceUnity AR Filter extension in your  Android or iOS app.
+
+## Understand the tech
+
+The FaceUnity AR Filter extension encapsulates the core APIs of the FaceUnity Nama SDK. By calling the <Link to="{{Global.API_REF_ANDROID_ROOT}}/class_irtcengine.html#api_setextensionproperty">setExtensionProperty</Link>
```
Comment (dengyaxi) [inline extensions-marketplace/develop/integrate/faceunity.mdx:14]: This sentence seems to be redundant. It breaks the flow

File: extensions-marketplace/develop/integrate/faceunity.mdx [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+
+<PlatformWrapper notAllowed="android">
+<PlatformWrapper notAllowed="ios">
+This extension is not provided for this platform.
+</PlatformWrapper>
+</PlatformWrapper>
```
Comment (dengyaxi) [inline extensions-marketplace/develop/integrate/faceunity.mdx:20]: Maybe it's better to move this message before "understand the tech"?

File: extensions-marketplace/develop/integrate/pinpoint.mdx [hunk 4]
```diff
@@ -0,0 +1,153 @@
+---
+title: "PinPoint"
```
Comment (dengyaxi) [inline extensions-marketplace/develop/integrate/pinpoint.mdx:2]: For consistency with the [Extensions Gallery](https://www.agora.io/en/agora-extensions-marketplace/), please change it to "Bose PinPoint". This is what's agreed by our partner

**Review by dengyaxi — CHANGES_REQUESTED** [review]

**Comment by dengyaxi** [comment]
Hi @billy-the-fish. Thank you for adding the platform filters. Nice work.

I have one more thing for discussion, though. "Implement an extension" in the quickstart title and top-level TOC sounds ambiguous. Can you change back to "create an extension"?

**Comment by dengyaxi** [comment]
One last thing. Could you reorder the Publish your extension to the end in "Implement an extension"? This would look more logical

File: extensions-marketplace/develop/integrate/banuba.mdx (hunk 1 above)
Comment (billy-the-fish) [reply extensions-marketplace/develop/integrate/banuba.mdx:17]: Nice spot. done.

File: extensions-marketplace/develop/integrate/faceunity.mdx (hunk 2 above)
Comment (billy-the-fish) [reply extensions-marketplace/develop/integrate/faceunity.mdx:14]: Done.

File: extensions-marketplace/develop/integrate/faceunity.mdx (hunk 3 above)
Comment (billy-the-fish) [reply extensions-marketplace/develop/integrate/faceunity.mdx:20]: :-), done.

File: extensions-marketplace/develop/integrate/pinpoint.mdx (hunk 4 above)
Comment (billy-the-fish) [reply extensions-marketplace/develop/integrate/pinpoint.mdx:2]: Done.

File: extensions-marketplace/develop/integrate/virtual-background.mdx [hunk 5]
```diff
@@ -0,0 +1,360 @@
+---
```
[the remark this answers is not in the document]
Comment (billy-the-fish) [reply extensions-marketplace/develop/integrate/virtual-background.mdx:1]: I will leave as is, discuss with Emily and co and sort this post release. 

Why? If this is part of VSDK, the doc needs to be rewritten to match the 4.x docs, or added to a doc. Also, it is a nice example to use in the discussion as to what an  extension is. If we use the extensions mechanism to add features, fine. But in that case, we have to put it into a story.

**Comment by billy-the-fish** [comment]
For some odd reason, this did not go into the content release last night. I have merged it to https://github.com/AgoraIO/Docs-Source/tree/449-content-release-8 which is tonights content release branch.

**Pull request merged**